### PR TITLE
Updated Documentation

### LINKED ISSUE #686: Add documentation for Container and Scrollable components
@ajmaln _a

cc: @karthiknmenon

### LINKED ISSUE #693: Toastr not working as expected
@yedhink 

* Styling for Toastr not mentioned in the neetoui/v2 doc.
```css
@import "react-toastify/dist/ReactToastify.min.css";
```

* React-Toastify is not installed with neeto-ui.

### PR DESCRIPTION
**Pull request by ajmaln**
- Fixes #693 
  - Added all peer dependency details including `react-toastify` to documentation
<br>

- Fixes #686 
  - Added stories with different variants for Container and Scrollable
  - Added documentation on props for Container and Scrollable

@ajmaln _a

### CODE REVIEW

**Comment by ajmaln** [comment]
@amaldinesh7 _a Please review.

cc: @karthiknmenon

File: README.md [hunk 1]
```diff
@@ -15,6 +15,35 @@ Starting `3.0.x`, neetoUI stylesheet has been separated from the bundle. To get
 @import "@bigbinary/neetoui/v2";
 ```
 
+**NeetoUI** has few peer dependencies which are required to use NeeetoUI properly. Install those peer dependencies using the below command.:
```
Comment (amaldinesh7) [inline README.md:18]: Please fix the typo `NeeetoUI`. Please remove the full stop and keep the colon

File: stories/Introduction.stories.mdx [hunk 2]
```diff
@@ -59,6 +59,35 @@ Starting `3.0.x`, neetoUI stylesheet has been separated from the bundle. To get
 @import "@bigbinary/neetoui/v2";
 ```
 
+**NeetoUI** has few peer dependencies which are required to use NeeetoUI properly. Install the peer dependencies using the below command.:
```
Comment (amaldinesh7) [inline stories/Introduction.stories.mdx:62]: Same here

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@ajmaln _a Please take a look at the comments.

**Comment by ajmaln** [comment]
@amaldinesh7 _a fixed the typo.

**Pull request merged**